docs-guides/source/mlmodel-utilities.md

```diff
@@ -98,3 +98,60 @@ input_data = [{'input_1': 1, 'input_2': 2}, {'input_1': 3, 'input_2': 3}]
 expected_output = [{'input_1': 2.5, 'input_2': 2.0}, {'input_1': 1.3, 'input_2': 2.3}]
 metrics = coremltools.utils.evaluate_transformer(scaler_spec, input_data, expected_output)
 ```
+
+
+```{eval-rst}
+.. index:: 
+    single: weight metadata
+```
+
+
+## Get Weights Metadata
+
+The  [`get_weights_metadata()`](https://apple.github.io/coremltools/source/coremltools.optimize.coreml.post_training_quantization.html#coremltools.optimize.coreml.get_weights_metadata) utility provides a convenient way to inspect the properties of the weights in the model. You may want to use it for debugging, or for identifying weights that are larger than a certain size, have a sparsity greater than a certain percentage, or have a certain number of unique elements. 
+
+For example, if you want to compare the weights of a model showing unexpected results with the weights of a model with predictable results, you can use `get_weights_metadata()` to get a list of all the weights with their metadata. Use the `weight_threshold` parameter to set which weights are returned. A weight is included in the resulting dictionary only if its total number of elements are greater than `weight_threshold`. 
+
+The metadata returned by the utility also offers information about the child ops the weight feeds into. The data returned by the API can then be used to customize the optimization of the model via the `ct.optimize.coreml` API. 
+
+### Using the Metadata 
+
+The  [`get_weights_metadata()`](https://apple.github.io/coremltools/source/coremltools.optimize.coreml.post_training_quantization.html#coremltools.optimize.coreml.get_weights_metadata) utility returns the weights metadata as a  [CoreMLWeightMetaData](https://apple.github.io/coremltools/source/coremltools.optimize.coreml.post_training_quantization.html#coremltools.optimize.coreml.CoreMLWeightMetaData) dictionary mapping each weight’s name to its metadata. The results are useful when constructing [`cto.OptimizationConfig`](https://apple.github.io/coremltools/docs-guides/source/optimizecoreml-api-overview.html#customizing-ops-to-compress).
```

Comment:
Just for clarification,
the returned type of `get_weights_metadata` is a dictionary that mapping string to `CoreMLWeightMetaData`.

However, the above doc string says:
`utility returns the weights metadata as a  [CoreMLWeightMetaData]`, will it be a little confusing at the first glimpes?